UnitTests/EncogWrapperTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using NUnit.Framework;
+using MotionRecognition;
+using Encog.Engine.Network.Activation;
+
+namespace UnitTests
+{
+	public class EncogWrapperTests
+	{
+
+		NetworkContainer container;
+
+		[SetUp]
+		public void Setup()
+		{
+			container = new NetworkContainer();
+			EncogWrapper.Instantiate(ref container);
+
+			EncogLayerSettings inputLayerSettings = new EncogLayerSettings();
+			inputLayerSettings.activationFunction = null;
+			inputLayerSettings.hasBias = true;
+			inputLayerSettings.neuronCount = 100;
+
+			EncogLayerSettings hiddenLayerOneSettings = new EncogLayerSettings();
+			hiddenLayerOneSettings.activationFunction = new ActivationElliott();
+			hiddenLayerOneSettings.hasBias = true;
+			hiddenLayerOneSettings.neuronCount = 100;
+
+			EncogLayerSettings outputLayerSettings = new EncogLayerSettings();
+			outputLayerSettings.activationFunction = new ActivationElliott();
+			outputLayerSettings.hasBias = false;
+			outputLayerSettings.neuronCount = 1;
+
+			EncogWrapper.AddLayer(ref container, ref inputLayerSettings);
+
+			EncogWrapper.AddLayer(ref container, ref hiddenLayerOneSettings);
+
+			EncogWrapper.AddLayer(ref container, ref outputLayerSettings);
+
+			EncogWrapper.finalizeNetwork(ref container);
+		}
+
+		[Test]
+		public void InstantiationTest()
+		{
+			Assert.IsNotNull(container.network);
+		}
+
+		[Test]
+		public void AddLayer()
+		{
+			var Layers = container.network.Structure.Flat.LayerCounts;
```

Comment:
In deze test, test je dus de AddLayer methodes die in setup aangeroepen worden?

Zet ze dan even in de test neer, in het geval dat de functie dan faalt, faalt de betreffend unit test ipv de setup. (in dit geval zou de setup falen en moet je als nog debuggen om te zien wat er mis gaat)